File_handling/coverage/lcov-report/base.css

```diff
@@ -0,0 +1,224 @@
+body, html {
```

Comment:
Add the whole coverage folder to gitignore

Reply:
Done